### ISSUE TITLE
ringer volume is announced when screen turns off or on.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#138]
```
What steps will reproduce the problem?
1. press the power button.
2. talk back says "screen off, ringer 100 percent."
3. now press the power button again, and talkback will say "time/notifications, 
ringer100 percent."

What is the expected output? What do you see instead?
talkback should announce ""screen off" or "time/notifications"

What version of the product are you using? On what operating system?
included talkback with android 4.1.1 on galaxy nexus and 3.1.4 beta 5 on nexus 7

Please provide any additional information below.


```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 21 Sep 2012 at 7:33

**Comment by GoogleCodeExporter** [comment]
```
after doing some more testing, the nexus 7 with talkback 3.1.4b5 does not say 
it when you lock the scren, just when you un lock the device, but the galaxy 
nexus says it when you lock and unlock the screen, I did have 3.1.4b5 on my 
galaxy nexus and it said it all of the time when I would go to lock and unlock 
the screen.
```

Original comment by `jmwalto...@gmail.com` on 21 Sep 2012 at 7:43

**Comment by GoogleCodeExporter** [comment]
```
Speaking the ringer volume on devices with telephony is the intended behavior 
on screen on and off.
```

Original comment by `caseybur...@google.com` on 15 Oct 2012 at 10:12
- Changed state: **Invalid**
